README.md

```diff
@@ -1,5 +1,19 @@
 # SPipes Editor UI
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
+  - exploration of the related filesystem structure where scripts are located
+  - represent the pipelines as graphs
+  - collapse/expand groups of nodes based on the source file they come from
```

Comment:
Maybe the following:

`collapse/expand groups of nodes based on the source file`